dateparser/conf.py

```diff
@@ -193,6 +195,12 @@ def check_settings(settings):
         'PREFER_LOCALE_DATE_ORDER': {
             'type': bool
         },
+        'DEFAULT_LANGUAGES': {
+            'type': list
+        },
+        'LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD': {
+            'type': float
```

Comment:
Hi @gavishpoddar, the `LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD` can take values between 0 and 1, but you are allowing any value.


You have to add here an "extra_check" to check the values:

```python
'LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD': {
    'type': float,
    'extra_check': _check_between_0_and_1
},

```

Then, you can define the `_check_between_0_and_1` (after the `_check_parsers` function):

```python
def _check_between_0_and_1(setting_name, setting_value):
    is_valid = 0 <= setting_value <= 1
    if not is_valid:
        raise SettingValidationError(
            '{} is not a valid value for {}. It can take values between 0 and '
            '1.'.format(
                setting_value, setting_name,
            )
        )
```

And finally add a test in `test_settings`:

```python
def test_check_settings_extra_check_confidence_treshold(self):
    with self.assertRaisesRegex(
        SettingValidationError,
        r'1.1 is not a valid value for '
        r'LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD. It can take values '
        r'between 0 and 1'
    ):
        DateDataParser(settings={'LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD': 1.1})
```

Reply:
Thanks Marc, changes are done